doc/release/1.9.0-notes.rst

```diff
@@ -70,60 +145,806 @@ New features
+``signal.convolve2d`` now raises an error
+- The ``spatial.distance`` now enforce that the input vectors are
+one-dimensional.
+- Removed ``stats.itemfreq``
+- Removed ``stats.median_absolute_deviation``
+- Removed ``n_jobs`` keyword argument and use of ``k=None`` from
+``kdtree.query``
+- Removed ``right`` keyword from ``interpolate.PPoly.extend``
+- Removed ``debug`` keyword from ``scipy.linalg.solve_*``
+- Removed class ``_ppform`` ``scipy.interpolate``
+- Removed BSR methods ``matvec`` and ``matmat``
+- Removed ``mlab`` truncation mode from ``cluster.dendrogram``
+- Removed ``cluster.vq.py_vq2``
+- Removed keyword arguments ``ftol`` and ``xtol`` from
+``optimize.minimize(method='Nelder-Mead')``
+- Removed ``signal.windows.hanning``
+- Removed LAPACK ``gegv`` functions from ``linalg``; this raises the minimally
+required LAPACK version to 3.7.1
+- Removed ``spatial.distance.matching``
+- Removed the alias ``scipy.random`` for ``numpy.random``
+- Removed docstring related functions from ``scipy.misc`` (``docformat``,
+``inherit_docstring_from``, ``extend_notes_in_docstring``,
+``replace_notes_in_docstring``, ``indentcount_lines``, ``filldoc``,
+``unindent_dict``, ``unindent_string``).
+- Removed ``linalg.pinv2``
 
 ******************************
 Backwards incompatible changes
 ******************************
 
+- Several `scipy.stats` functions now convert ``np.matrix`` to ``np.ndarray``s
+before the calculation is performed. In this case, the output will be a scalar
+or ``np.ndarray`` of appropriate shape rather than a 2D ``np.matrix``.
+Similarly, while masked elements of masked arrays are still ignored, the
+output will be a scalar or ``np.ndarray`` rather than a masked array with
+``mask=False``.
+- The default method of ``scipy.optimize.linprog`` is now ``'highs'``, not
+``'interior-point'`` (which is now deprecated), so callback functions and some
+options are no longer supported with the default method.
+- For `scipy.stats.combine_pvalues`, the sign of the test statistic returned
+for the method ``"pearson"`` has been flipped so that higher values of the
+statistic now correspond to lower p-values, making the statistic more
+consistent with those of the other methods and with the majority of the
+literature.
+- `scipy.linalg.expm` due to historical reasons was using the sparse
+implementation and thus was accepting sparse arrays. Now it only works with
+nDarrays. For sparse usage, `scipy.sparse.linalg.expm` needs to be used
+explicitly.
+- The definition of `scipy.stats.circvar` has reverted to the one that is
+standard in the literature; note that this is not the same as the square of
+`scipy.stats.circstd`.
+- Remove inheritance to `QMCEngine` in `MultinomialQMC` and
+`MultivariateNormalQMC`. It removes the methods `fast_forward` and `reset`.
+- Init of `MultinomialQMC` now require the number of trials with `n_trials`.
+Hence, `MultinomialQMC.random` output has now the correct shape ``(n, pvals)``.
```

Comment:
Email [here](https://mail.python.org/archives/list/scipy-dev@python.org/thread/QEMS6VZSBHTRB6OVSDBULJNNOE7PFAOV/#QEMS6VZSBHTRB6OVSDBULJNNOE7PFAOV), discussion [here](https://github.com/scipy/scipy/pull/15945). This was definitely a bug.